proposed/http-message.md

```diff
@@ -111,8 +111,8 @@ request or response.
 In order to accommodate headers with multiple values yet still provide the
 convenience of working with headers as strings, headers can be retrieved from
 an instance of a `MessageInterface` as an array or a string. Use the
-`getHeader()` method to retrieve a header value as a string containing all
-header values of a case-insensitive header by name concatenated with a comma.
+`getHeader()` method to retrieve a string containing concatenated
+header values of a case-insensitive header by name.
```

Comment:
The original verbiage indicated that the value WILL be returned as a comma-concatenated string. This was done to enforce a common expectation and signature from what is returned by the method. The implication is that if you know a given header CAN NOT be concatenated in this way, you will use the `getHeaderLines()` method and concatenate them manually. This also ensures that implementations do not have to be aware of every single header type, particularly as RFC 7230 allows for custom header types, which may have their own rules about concatenation.